magefiles/magefile.go

```diff
@@ -61,22 +61,6 @@ func buildLdflags() (string, error) {
 
 type Tool mg.Namespace
 
-// Aqua installs aqua if not installed
-func (Tool) Aqua() error {
```

Comment:
I think we should keep the aqua install feature. 
It will help with installing tinyGo on a new PC.

Reply:
Or better to add aqua into `go.mod`

Reply:
I added aqua and reverted it due to unknown errors. Let me add it back and see how it goes.

Reply:
https://github.com/aquasecurity/trivy/pull/8493/commits/ac80c7f179e27cc112757c191cec17209e868b9f